# QuadPosition Message

# is the body found by the mocap system
bool found_body

# Give the position of the quad in the Qualisys frame
float64 x
float64 y
float64 z
float64 pitch
float64 roll
float64 yaw
